src/ui/map/Map.svelte

```diff
@@ -17,6 +19,19 @@
 
   let container;
 
+  $: {
+    if ($selectedGeography.lad) {
+      if (ladBoundsLookup[$selectedGeography.lad]) {
```

Comment:
Petty but these could be joined with logical and.

Reply:
Done